java/src/main/java/api/example/java/model/Field.java

```diff
@@ -0,0 +1,50 @@
+package api.example.java.model;
+
+public class Field {
+    private String id;
+    private String name;
+    private String boundaryId;
+    private String resourceOwnerId;
+    private Parent parent;
```

Comment:
Does the parent here represent the farm this field belongs to ? If it is, it might make sense to rename Parent to Farm.

Reply:
Just named as it is named as output of the API